src/main/java/com/courtcanva/ccfranchise/model/Staff.java

```diff
@@ -0,0 +1,75 @@
+package com.courtcanva.ccfranchise.model;
+
+import com.courtcanva.ccfranchise.constants.StaffRole;
+import com.courtcanva.ccfranchise.constants.VerifyStatus;
+import lombok.*;
```

Comment:
Importing all(*) is not allowed in our team.

Reply:
already fix it